CesiumGltf/test/TestPropertyView.cpp

```diff
@@ -3058,3 +3187,215 @@ TEST_CASE("String Array PropertyView") {
     REQUIRE(view.status() == PropertyViewStatus::ErrorInvalidNoDataValue);
   }
 }
+
+TEST_CASE("Enum PropertyView") {
```

Comment:
Sorry for the nitpick, but can this be put under the `String PropertyView` tests? Just so the non-array tests are grouped together.